config/webpack/environment.js

```diff
@@ -1,3 +1,11 @@
-const { environment } = require('@rails/webpacker')
-
+const { environment } = require("@rails/webpacker")
+const webpack = require("webpack");
+environment.plugins.append(
+"Provide",
```

Comment:
sai indent rồi em, fix lại file này
```suggestion
  "Provide",
```